tests/callbacks/test_tqdm_progress_bar.py

```diff
@@ -618,3 +618,32 @@ def test_step(self, batch, batch_idx):
 
     trainer.test(model, verbose=False)
     assert pbar.calls["test"] == []
+
+
+@RunIf(standalone=True)
```

Comment:
I tried running 
`python -m pytest tests/callbacks/test_tqdm_progress_bar.py::test_tqdm_progress_bar_disabled_when_not_rank_zero tests/checkpointing/test_model_checkpoint.py::test_model_checkpoint_score_and_ckpt`
(after removing the standalone decorator)

And had no issues. If standalone helps it's probably because there's an unwanted interaction between tests.

This should be investigated before merging this PR

Reply:
yes, the tests work locally. the problem is only in the CI. It does seem like there's an unwanted interaction between tests. I have already spent many hours investigating this, and narrowed the problem down to the`@mock.patch` decorator. I tried changing it to `with mock.patch ...` inside the test, but that didn't help either. however when I remove the `mock.patch` completely, the checkpointing test passes (in the CI).

At this point I don't know what to do next. One option is we could merge this without the test, since this change would still be a strict improvement from the current situation.

Reply:
If it's consistent in the CI, we can ssh into it and debug there

Just needs to add this

```yaml
    - name: Setup tmate session
      if: always()
      uses: mxschmitt/action-tmate@v3
```

To the problematic job